docs/docs/external-volumes.md

```diff
@@ -84,7 +84,7 @@ You can specify an external volume in your Marathon app definition. [Learn more
       "instances": 1,
       "cpus": 0.1,
       "mem": 32,
-      "cmd": "/usr/bin/tail -f /dev/null",
+      "cmd": "mkdir my-test-vol && touch my-test-vol/test.txt && ls my-test-vol/",
```

Comment:
@jdef I wasn't able to get the `touch` part of this command to work without first creating the directory. Assuming it was some kind of error on my part -- retesting.